pipelines/build/common/build_base_file.groovy

```diff
@@ -141,6 +142,7 @@ class Builder implements Serializable {
                 ADOPT_BUILD_NUMBER: adoptBuildNumber,
                 ENABLE_TESTS: enableTests,
                 ENABLE_INSTALLERS: enableInstallers,
+                ENABLE_SINGER : enableSigner,
```

Comment:
`ENABLE_SINGER` -> `ENABLE_SIGNER`

Reply:
Yes. Could you help me trigger tests again and see how it goes? Thanks for pointing out the typo.

Reply:
Once the code has been updated, ping me and I'll trigger them for you 🙂

Reply:
@M-Davies  Updated, please have another look. Thanks.